### PR TITLE
Detect ama errors

### PR DESCRIPTION
**Pull request by sosukesuzuki**
we open Boostnote, when disconnection Internet. Then, console shows many error message(like below the image).
So, I detect internet disconnection, and stop sending event.

### CODE REVIEW

File: browser/main/lib/AwsMobileAnalyticsConfig.js [hunk 1]
```diff
@@ -34,8 +34,15 @@ function convertPlatformName (platformName) {
   }
 }
 
+function getSendEventCond () {
+  const isDev = process.env.NODE_ENV !== 'production'
+  const isDisable = !ConfigManager.default.get().amaEnabled
+  const isOffline = !window.navigator.onLine
+  return isDev || isDisable || isOffline
```
Comment (sota1235) [inline browser/main/lib/AwsMobileAnalyticsConfig.js]: good work
why don't you use this function on [this line](#diff-7091051d0797267a05e32dd7ac21ad56R10)?

Comment (sosukesuzuki) [reply browser/main/lib/AwsMobileAnalyticsConfig.js]: thanks for your commenting!
I fixed it on b7e43d9

**Comment by kazup01** [comment]
Waiting for your review. @sota1235

**Comment by kazup01** [comment]
Merged

**Pull request merged**